frame/treasury/src/lib.rs

```diff
@@ -222,6 +259,23 @@ pub mod pallet {
 		OptionQuery,
 	>;
 
+	/// Proposals that have been made.
+	#[pallet::storage]
+	#[pallet::getter(fn pending_payments)]
+	pub type PendingPayments<T: Config<I>, I: 'static = ()> = CountedStorageMap<
+		_,
+		Twox64Concat,
+		ProposalIndex,
```

Comment:
I kept it as ProposalIndex because of https://github.com/paritytech/substrate/pull/13607#discussion_r1137134274 
As opposed to introducing a new type.

Reply:
Yeah, but why the alternative would be ProposalCount?
there are no proposals for new flow, but pending payments

Reply:
At the time when that comment was made, the count was used for the Proposals storage map. https://github.com/paritytech/substrate/pull/13607/commits/a0275942d2e391e6a9fc3181de95faca915d8d36#diff-21571fa193fececabaa733a8fea9e3c6478e1cdcb86f58e3f325418b2555e81aR237 

I could introduce `PendingPaymentsCount` type, but the type going to be a u32 as well, just like ProposalIndex.  But yeah, it's likely a good idea for clarity in the code. Though i'll call it `PendingPaymentsIndex` for consistency.